core/domain/html_cleaner.py

```diff
@@ -151,13 +149,22 @@ def get_image_filenames_from_html_strings(html_strings: List[str]) -> List[str]:
 
     for rte_comp in all_rte_components:
         if 'id' in rte_comp and rte_comp['id'] == 'oppia-noninteractive-image':
-            filenames.append(
-                rte_comp['customization_args']['filepath-with-value'])
+            # Here we use cast because the above 'if' condition forces
+            # customization_args to have type str.
```

Comment:
This is wrong, it forces the 'filepath' customization arg to have type str.

Reply:
Fixed